vpinball/vpinball
# feat: snap package

Work in progress

~~The fact that most repos require latest cmake is causing some issues~~

To test this 

Make sure you have the required tools
https://snapcraft.io/docs/create-a-new-snap#heading--setup


```
snapcraft
sudo snap install ./vpinball_10.8.1-git_amd64.snap --dangerous --devmode
```
